Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -172,13 +172,31 @@ public struct TestCommand: AsyncParsableCommand, LogConfigurableCommand {
     )
     var generateOnly: Bool = false
 
+    @Flag(
+        name: .long,
+        help: "When passed, run the tests without building.",
+        envKey: .testWithoutBuilding
+    )
+    var withoutBuilding: Bool = false
+
+    @Flag(
+        name: .long,
+        help: "When passed, build the tests, but don't run them",
+        envKey: .testBuildOnly
+    )
+    var buildOnly: Bool = false
```

Comment:
We should skip trying to store selective test results when this flag is passed.

Reply:
Great point, I've fixed that